otter/models/cass.py

```diff
@@ -444,11 +448,27 @@ def _jsonloads_data(raw_data):
     return data
 
 
+def _group_status(status, deleting):
+    if deleting:
+        return ScalingGroupStatus.DELETING
+    else:
+        # TODO: #1304
```

Comment:
Non-blocking nitpick - since this would nice to have, but is already tracked by an issue, can this comment be removed?  Alternately can it instead be moved next to the 'DISABLED' part of these conditionals?